lib/perl/Genome/Model/Build/Command/Export.pm

```diff
@@ -97,6 +120,14 @@ sub execute {
 
     $find->($export_directory);
 
+    if($self->create_tarball){
+        my $tarball = File::Spec->join($self->target_export_directory, "build" . $self->build->id . ".tar");   
+        my $cmd = "tar -cf $tarball -C $tempdir build" . $self->build->id;
+        system("$cmd") and do {
```

Comment:
I'd prefer to see `Genome::Sys->shellcmd` over `system`.  Or maybe `Genome::Sys->tar`:
https://github.com/genome/genome/blob/cd160fd03c34189ea7a362a8afc8e395a9843fc4/lib/perl/Genome/Sys.pm#L740

(Using `-C` could be a nice improvement to that implementation, and it (or here) could use the array form of specifying the command!)